alphagov/paas-tech-docs
# Merge PostgreSQL and MySQL into Database Services

## What

Added in MongDB content and updated the Using SERVICE section to apply to all backing services.

## Describe what you have changed and why.

1. Added MongoDB reference to the "Deploy a backing or routing service" > "Services and plans"

2. Grouped High Availability (HA) Plans under another title; "Failover process"
- Specified that HA plans are for Postgres and MySQL
- Added MongoDB content for HA Plans

3. Changed Using PostgreSQL section to Using Database Services, and specified that it applies to PostgreSQL, MySQL and MongoDB. This removes the duplicated content between Using MySQL and Using PostgreSQL and includes MongoDB where appropriate. If this is approved, the separate Using MySQL section would be deleted.

## How to review

- Check if the proposed removal of duplicated content makes sense from a user journey perspective.
- Check if any other information is necessary to enable a user to setup the MongDB backing service.
- Check if the added content on MongoDB HA plans is correct.

## Who can review

Piotr Komborski
Holly Challenger
